docs/changelog.txt

```diff
@@ -43,6 +43,7 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 - `tiletypes-here`, `tiletypes-here-point`: add --cursor and --quiet options to support non-interactive use cases
 
 ## Documentation
+- `quickfort`: Dreamfort blueprint set improvements: extensive revision based on playtesting and feedback. includes updated supporting ``onMapLoad.init`` file and automation orders JSON files. see full changelog at https://github.com/DFHack/dfhack/pull/1921
```

Comment:
Is this the right section? Or do you want it under "Misc Improvements", maybe? Or split? (Asking because it looks like a lot more than just documentation changes)

Reply:
I'm, um, not sure that ended up in the documentation section. I intended it for `Misc Improvements`. Slip of the arrow keys? Unintended consequence of a rebase? Dunno. But will fix.

What are your thoughts on where the supporting files are stored? Would the automation orders, sample `onMapLoad.init`, and sample professions files be better off distributed with DFHack? I designed them to be useful beyond just Dreamfort, and I'm starting to think they would benefit from version control, but that doesn't mean that they have to be part of the DFHack package.

If we do add them to the DFHack repo, I'm not suggesting that they be loaded automatically. The `onMapLoad.init` file is too opinionated for that, especially with the autobutcher settings. They could be added to an `extras` directory or something, though.

Reply:
It makes sense to me for them to be tracked. Unsure if there's a good existing location for them, though. `hack/examples` maybe? We have a bit of a special case with example raws currently, with `plugins/raw/*.txt` getting installed to `hack/raw/`, but this wouldn't really fall into the same category (just noting something similar we have set up already).

Reply:
I like the idea of `hack/examples`.

Reply:
That works for me, as long as we make it clear that they are actually ready-to-use, not just toy examples or syntax examples

Reply:
Ok, I'll merge this PR as-is and address the additional files in a new PR